cli/src/semgrep/app/scans.py

```diff
@@ -233,7 +234,7 @@ def report_findings(
         parse_rate: ParsingData,
         total_time: float,
         commit_date: str,
-        lockfile_scan_info: Dict[str, int],
+        lockfile_dependencies: Dict[str, List[FoundDependency]],
```

Comment:
Now that we have full dependency list here, easy to send data once app side toggle is done.